### DISCUSSION TITLE
[Feature] Support for ViT-B-32-256 / datacomp_s34b_b86k

### DISCUSSION THREAD

**Discussion by imohitkr** [immich-app/immich#11586] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi,
I was told in the docs to mention here if we require support for new models. I was checking the Open CLIP repo and it seems that the FLOPs requirement for ViT-B-32-256/ datacomp_s34b_b86k is same as the default one used within Immich. But this one offers 8 percentage point higher performance on all the datasets. It would be good to switch to this or at least give option to switch to this. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
It's outclassed by `ViT-B-32/laion2b_s34b_b79k`, which we do offer. I'm thinking of changing the default to this model.

**Reply by imohitkr** [reply]
Yeah, I switched to this. But still the `ViT-B-32-256/ datacomp_s34b_b86k` is 4 points higher. 
https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv

**Reply by mertalev** [reply]
The more accurate spreadsheet is for [retrieval](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_retrieval_results.csv), specifically image retrieval. The datacomp model is the best at its FLOP class in performing tasks, but it isn't as good at retrieval.

**Reply by imohitkr** [reply]
Okay, that makes sense. Thanks, we can mark this request as closed. Thanks for responding promptly.
